src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -86,16 +86,18 @@
     context "a valid request" do
       before do
         post :delete_request, params: { project: target_project, package: target_package, description: "delete it!" }
-        @bs_request = BsRequest.joins(:bs_request_actions).
+      end
+
+      subject do
+        BsRequest.joins(:bs_request_actions).
           where("bs_request_actions.target_project=? AND bs_request_actions.target_package=? AND type=?",
                 target_project.to_s, target_package.to_s, "delete"
                ).first
       end
 
-      it { expect(response).to redirect_to(request_show_path(number: @bs_request)) }
-      it { expect(flash[:success]).to match("Created .+repository delete request #{@bs_request.number}") }
-      it { expect(@bs_request).not_to be nil }
-      it { expect(@bs_request.description).to eq("delete it!") }
+      it { expect(response).to redirect_to(request_show_path(number: subject)) }
```

Comment:
Should also be ``subject.number``, no?
(It probably works because ``number == id`` in this case)

Reply:
@ChrisBr Nah, we [changed](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/bs_request.rb#L339) that.